.gitignore

```diff
@@ -0,0 +1,29 @@
+env
+.env
```

Comment:
I think `*.env` and `!sample.env` would be better here, then we don't track _any_ `.env` files, including default or other configs that might be in the directory.